### PR TITLE
Docs: Update time range controls

### LINKED ISSUE #21886: Docs feedback: /reference/timerange.md
This page is showing the old time range selector

### PR DESCRIPTION
**Pull request by oddlittlebird**
**What this PR does / why we need it**: The Time range controls topic was out of date and incomplete. This PR updates, rewrites, and moves the file.

**Which issue(s) this PR fixes**:

Fixes #21886

**Special notes for your reviewer**:
Images are in this PR: https://github.com/grafana/website/pull/1486
Which is currently stuck because CircleCI is apparently very, very broken atm.

### CODE REVIEW

File: docs/sources/dashboards/time-range-controls.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+<img class="no-shadow" src="/img/docs/time-range-controls/common-time-controls-7-0.png" max-width="700px">
+
+The options are defined below.
+
+### Current time range
+
+The current time range, also called the _time picker_, shows the time range currently displayed in the dashboard or panel you are viewing.
+
+Hover your curser over the field to see the exact time stamps in the range and their source (such as the local browser).
```
Comment (marefr) [inline docs/sources/dashboards/time-range-controls.md:46]:
```suggestion
Hover your cursor over the field to see the exact time stamps in the range and their source (such as the local browser).
```

File: docs/sources/dashboards/time-range-controls.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+- Today so far
+- This week so far
+- This month so far
+
+### Absolute time range
+
+Set an absolute time range one of two ways:
+
+- Type values into the **From** and **To** fields. You can type exact time values or relative values, such as `now-24h`, and then click **Apply time range**.
+- Click in the **From** or **To** field. Grafana displays a calender. Click the day or days you want to use as the current time range and then click **Apply time range**.
```
Comment (marefr) [inline docs/sources/dashboards/time-range-controls.md:74]:
```suggestion
- Click in the **From** or **To** field. Grafana displays a calendar. Click the day or days you want to use as the current time range and then click **Apply time range**.
```

File: docs/sources/dashboards/time-range-controls.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+Grafana provides several ways to manage the time ranges of the data being visualized, both at the dashboard level and the panel level. 
+
+This page describes supported time units and relative ranges, the common time controls, dashboard-wide time settings, and panel-specific time settings.
+
+## Time units and relative ranges
+
+The following time units are supported: `s (seconds)`, `m (minutes)`, `h (hours)`, `d (days)`, `w (weeks)`, `M (months)`, and `y (years)`. 
+
+The minus operator allows you to step back in time, relative to now. If you wish to display the full period of the unit (day, week, month, etc...), append `/$unit` to the end.
```
Comment (marefr) [inline docs/sources/dashboards/time-range-controls.md:22]: Not sure, `/$unit`makes sense or if you're using that in other places. Suggest:

```suggestion
The minus operator allows you to step back in time, relative to now. If you wish to display the full period of the unit (day, week, month, etc...), append `/<time unit>` to the end.
```

File: docs/sources/dashboards/time-range-controls.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
```
Comment (marefr) [inline docs/sources/dashboards/time-range-controls.md:23]: I think a paragraph which explains that the plus operator can be used to step forward in time relative to now should also be added. Maybe not used by everyone, but there are users who uses this feature to look at predicted data in the future for example.

File: docs/sources/dashboards/time-range-controls.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+### Current time range
+
+The current time range, also called the _time picker_, shows the time range currently displayed in the dashboard or panel you are viewing.
+
+Hover your curser over the field to see the exact time stamps in the range and their source (such as the local browser).
+
+<img class="no-shadow" src="/img/docs/time-range-controls/time-picker-7-0.png" max-width="300px">
+
+To change the current time range, click the down arrow next to the current time range. You can change the current time using a _relative time range_, such as the last 15 minutes, or an _absolute time range_, such as `now-45h`.
```
Comment (marefr) [inline docs/sources/dashboards/time-range-controls.md:50]: The time range plus down arrow is "clickable". Also the absolute time range fields shows the relative time selected (if a relative time is selected), but an absolute time range is for example `2020-05-14 00:00:00 to 2020-05-15 23:59:59`. Not sure how to easily explain this 

```suggestion
Click on the current time range to change the time range. You can change the current time using a _relative time range_, such as the last 15 minutes, or an _absolute time range_, such as `2020-05-14 00:00:00 to 2020-05-15 23:59:59`.
```

File: docs/sources/dashboards/time-range-controls.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+- Type values into the **From** and **To** fields. You can type exact time values or relative values, such as `now-24h`, and then click **Apply time range**.
+- Click in the **From** or **To** field. Grafana displays a calender. Click the day or days you want to use as the current time range and then click **Apply time range**.
+
+This section also displays recently used absolute ranges.
+
+### Zoom out (Cmd+Z or Ctrl+Z)
+
+Click the **Zoom out** icon to view a larger time range in the dashboard or panel visualization.
+
+### Zoom in (not shown in UI)
```
Comment (marefr) [inline docs/sources/dashboards/time-range-controls.md:82]: Don't really understand this. Are you referring to zooming in to a time range in the graph panel?

File: docs/sources/dashboards/time-range-controls.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+By default, Grafana does not automatically refresh the dashboard. Queries run on their own schedule according to the panel settings. However, if you want to to regularly refresh the dashboard, then click the down arrow next to the **Refresh dashboard** icon and then select a refresh interval.
+
+## Dashboard time settings
+
+Time settings are saved on a per-dashboard basis.
+
+To access the dashboard time settings, click the **Dashboard settings** (gear) icon at the top of the screen. The settings are in the **Time Options** section of the General tab.
+
+- **Timezone -** Specify the local time zone of the service or system that you are monitoring. This can be helpful when monitoring a system or service that operates across several time zones.
+  - **Default -**: The default selected time zone for the user profile or organization is used. If no time zone is specified for the user profile or the organization, then Grafana uses local browser time.
```
Comment (marefr) [inline docs/sources/dashboards/time-range-controls.md:99]:
```suggestion
  - **Default -**: The default selected time zone for the user profile, team or organization is used. If no time zone is specified for the user profile, a team the user is a member of or the organization, then Grafana uses local browser time.
```

File: docs/sources/dashboards/time-range-controls.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+## Panel time overrides and timeshift
+
+In [Query options]({{< relref "queries.md#query-options" >}}), you can override the relative time range for individual panels, causing them to be different than what is selected in the dashboard time picker in the upper right. This allows you to show metrics from different time periods or days at the same time.
+
+## Control the time range using a URL
+
+Time range of a dashboard can be controlled by providing following query parameters in the dashboard URL:
+
+- `from` - defines lower limit of the time range, specified in ms epoch
```
Comment (marefr) [inline docs/sources/dashboards/time-range-controls.md:114]:
```suggestion
- `from` - defines lower limit of the time range, specified in ms epoch or [relative time]({{< relref "#relative-time-range" >}})
```

File: docs/sources/dashboards/time-range-controls.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+- `to` - defines upper limit of the time range, specified in ms epoch
```
Comment (marefr) [inline docs/sources/dashboards/time-range-controls.md:115]:
```suggestion
- `to` - defines upper limit of the time range, specified in ms epoch or [relative time]({{< relref "#relative-time-range" >}})
```

**Review by marefr — CHANGES_REQUESTED** [review]
Great. Left some minor comments/suggestions. Two spelling mistakes should resolve the build failure.

File: docs/sources/dashboards/time-range-controls.md (hunk 4 above)
Comment (oddlittlebird) [reply docs/sources/dashboards/time-range-controls.md:25]: Added

File: docs/sources/dashboards/time-range-controls.md (hunk 6 above)
Comment (oddlittlebird) [reply docs/sources/dashboards/time-range-controls.md:84]: This is literally the answer I got when I asked "how do you zoom in".

Comment (oddlittlebird) [reply docs/sources/dashboards/time-range-controls.md:84]: Updated content to clarify about the graph panel

**Review by marefr — APPROVED** [review]
LGTM :star2:

File: docs/sources/dashboards/time-range-controls.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
++++
+title = "Time range controls"
+keywords = ["grafana", "dashboard", "documentation", "time range"]
+type = "docs"
+[menu.docs]
+aliases = ["/docs/grafana/latest/reference/timerange/"]
```
Comment (torkelo) [inline docs/sources/dashboards/time-range-controls.md:6]: this alias is not working

**Pull request merged**